pypeit/reduce.py

```diff
@@ -555,6 +553,9 @@ def find_objects_pypeline(self, image, std=False, ir_redux=False, std_trace = No
         SHOW_TRACE:  bool
           Generate QA  showing traces identified. Requires an open ginga RC modules window
 
+        INTERACTIVE:  bool
```

Comment:
Can we remove this from the docs since the interactive feature is not there anymore.

Reply:
I thought I caught them all - thanks for catching that!